DBMS/Assignment1/Assign3.sql

```diff
@@ -0,0 +1,76 @@
+/* 
+Insert sample data in LIS tables by using SQL files.
+*/
+
+INSERT INTO Members VALUES  (1001, 'rajiv','sector-8','sector-10','M'),
+                            (1002, 'sanju','malviyanager','sitapura','F'),
+                            (1003, 'sanjiv','jagatpura','mansroavar','M'),
+                            (1004, 'rajshree','india gate','goner','F');
+						
+INSERT INTO Publishers VALUES (501,'Tata'),
+                              (502,'New world'),
+                              (503,'big rock');
+
+INSERT INTO Subjects VALUES (51,'arts'),
+                            (52,'science'),
+                            (53,'agriculture');
+							   
+INSERT INTO Titles VALUES (201,'love of Arts', 51,501),
+                          (202,'science world', 52,501),
+                          (203,'agriculture for you', 51,502);
+						
+INSERT INTO Books VALUES (101,201, '2013-01-01',500,'available'),
```

Comment:
in books table, second column type is of date. Have you verify that it works properly?